ecl/resource_ecl_vna_appliance_v1_mock_test.go

```diff
@@ -427,9 +429,16 @@ resource "ecl_vna_appliance_v1" "appliance_1" {
 		ip_address = "192.168.1.50"
 	}
 
+    initial_config = {
+        format = "set"
+        data = "c2V0IGludGVyZmFjZXMgZ2UtMC8wLzAgZGVzY3JpcHRpb24gc2FtcGxl"
+    }
```

Comment:
[MUST] スペースではなくタブでインデントしてください

```suggestion
	initial_config = {
		format = "set"
		data = "c2V0IGludGVyZmFjZXMgZ2UtMC8wLzAgZGVzY3JpcHRpb24gc2FtcGxl"
	}
```

Reply:
fix